docs/src/examples/rawkv.md

```diff
@@ -0,0 +1,51 @@
+# RawKV
+
+Below is the basic usages of RawKV. See [API document] to see a full list of methods available.
```

Comment:
could you provide a step-by-step guide about:

- how to set up a new maven project
- how to add java client dependency in the project
- how to program with java client APIs
- how to pass param to the application to replace the pd address and adjust the slow log threshold

except for the codes, please highlight the principles about using tikv txnkv API, for example:
- when to create and close a new TiSession?
- when to create and close a new Client?